package.json

```diff
@@ -11,31 +11,30 @@
     "vow-fs": "^0.3.4"
   },
   "devDependencies": {
-    "bem-components": "bem/bem-components#v2.1.0",
```

Comment:
Libs were moved to `bower.json` as that's the only way to get dependencies automatically. E.g. `bem-components` knows which `bem-core` version it depends on (still not right now as we released `bem-core` just today and haven't yet updated `bem-components`).